egs/wsj/s5/local/wsj_format_local_lms.sh

```diff
@@ -63,6 +206,32 @@ gunzip -c $lm_srcdir_3g/lm_unpruned.gz | \
 gunzip -c $lm_srcdir_4g/lm_unpruned.gz | \
   arpa2fst --disambig-symbol=#0 \
            --read-symbol-table=$lang/words.txt - data/lang${lang_suffix}_test_bd_fg/G.fst || exit 1;
+
+if [ $CLASSES == 1 ]; then
+  G_FST=data/lang${lang_suffix}_test_bd_fg/G.fst
+  G_WCLASS_FST=data/lang${lang_suffix}_test_bd_fg/G_wclass.fst
+  G_NO_WCLASS_FST=data/lang${lang_suffix}_test_bd_fg/G_no_wclass.fst
+  G_TMP_FST=data/lang${lang_suffix}_test_bd_fg/G_tmp.fst
+
+  cp $G_FST $G_WCLASS_FST
+
+  # go through all word classes in 'word_class_list.txt'
```

Comment:
I added two new scripts which do what you suggested:
1. local/wclass/create_wclass_SLMs.sh (which creates word-class sub-language models)
2. local/wclass/embed_wclass_SLMs.sh (which embeds word-class sub-language models in the overall language model G.fst)